lib/livebook_web/live/session_live/cell_component.ex

```diff
@@ -112,6 +112,13 @@ defmodule LivebookWeb.SessionLive.CellComponent do
             <%= remix_icon("arrow-up-s-line", class: "text-xl") %>
           </button>
         </span>
+        <span class="tooltip top" aria-label="Capture code snippet">
+          <button class="icon-button"
+            data-element="capture-code-snippet"
+            phx-value-cell_id="<%= @cell_view.id %>">
```

Comment:
Nit: we don't need `phx-value` here as it's handled on the client.

Reply:
Good one!